cmake/extensions.cmake

```diff
@@ -33,3 +33,21 @@ function(add_overlay_config image overlay_file)
   endif()
 endfunction()
 
+# Add a partition manager configuration file to the build.
+# Note that is only one image is included in the build,
+# you must set CONFIG_PM_SINGLE_IMAGE=y for the partition manager
+# configuration to take effect.
+function(add_partition_manager_config config_file)
+  get_filename_component(pm_path ${config_file} REALPATH)
```

Comment:
Should there be a check that this file actually exists ?
Or might it be generated during build (which means we cannot do much here)

Reply:
It should exist, added a check